spanner/spanner_snippets/spanner/spanner_cancel_backup_create.go

```diff
@@ -39,7 +46,15 @@ func cancelBackup(w io.Writer, db, backupID string) error {
 
 	expireTime := time.Now().AddDate(0, 0, 14)
 	// Create a backup.
-	op, err := adminClient.StartBackupOperation(ctx, backupID, db, expireTime)
+	request := adminpb.CreateBackupRequest{
```

Comment:
This is usually just `req` for samples.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#common-identifiers

Reply:
Fixed here and the other edited samples.